include/z64player.h

```diff
@@ -560,15 +570,11 @@ typedef struct Player {
     /* 0x06A8 */ Actor*     unk_6A8;
     /* 0x06AC */ s8         unk_6AC;
     /* 0x06AD */ u8         unk_6AD;
-    /* 0x06AE */ u16        unk_6AE;
-    /* 0x06B0 */ s16        unk_6B0;
+    /* 0x06AE */ u16        rotsNoRestFlags; // See `NOREST_ROT_` macros. If its flag isn't set, a rot steps to a rest value.
+    /* 0x06B0 */ s16        upperLimbYawSecondary;
```

Comment:
```suggestion
    /* 0x06B0 */ s16        upperLimbYawOffset;
```
Maybe?

Reply:
why not

"secondary" is because
```c
if (!(this->rotsNoRestFlags & NOREST_ROT_UPPER_Y)) {
    if (this->upperLimbYawSecondary != 0) {
        Player_ApproachZeroBinang(&this->upperLimbYawSecondary);
    } else {
        Player_ApproachZeroBinang(&this->upperLimbRot.y);
    }
}
```
, it steps this one to 0 before the main one